service/proxy/router/router_bucket.go

```diff
@@ -45,7 +45,7 @@ func (r *router) createBucket(req *http.Request) (resp *http.Response, task *tas
 	reqBody := createBucketConfiguration{}
 	err = s3client.ExtractReqBody(req, &reqBody)
 	if err != nil && !errors.Is(err, io.EOF) {
-		zerolog.Ctx(ctx).Err(err).Msg("unable to unmarshal createBucketConfiguration request body")
+		zerolog.Ctx(ctx).Warn().Err(err).Msg("unable to unmarshal createBucketConfiguration request body")
```

Comment:
I'm not sure, but it seems to make no sense to call `Warn()` and then `Err(err)`.

Reply:
it is ok, `.Err(err)` just adds error to json log `error` property. I've changed level to `warn` because this error is not critical.